test/unit/exception_handling_test.rb

```diff
@@ -140,12 +141,13 @@ def self.notify(data)
       ExceptionHandling.custom_data_hook = nil
     end
 
-    should "support a log_error hook and pass exception data to it" do
+    should "support a log_error hook and pass exception data and treat like warning to it" do
```

Comment:
Seems like you should have a test that uses the non-default value to ensure the plumbing is correct.

Reply:
done